src/firefly/html/firefly-dev.html

```diff
@@ -245,9 +245,9 @@
     <script  type="text/javascript" src="firefly_loader.js"></script>
 </head>
 
-<body style="margin: 0px;">
+<body style="margin: 0">
 <!-- attached location for firefly app -->
-<div id='app'/>
+<div id='app' style="width:100dvw;height: 100dvh"/>
```

Comment:
Just curious, what's benefit of dvw over vw?

Also, is the browser support roughly same?

Reply:
Although I've tested it with Chrome and Safari, it's a good point.  I'll change it to `vh` instead.  I don't think it'll make a different in our case.